docs/src/main/paradox/index.md

```diff
@@ -33,6 +34,7 @@ Various parts of Akka management can be used together for deploying Akka Cluster
   - [Akka Discovery Methods](discovery/index.md)
   - [Akka Cluster Management (HTTP)](cluster-http-management.md)
   - [Akka Cluster Management (JMX)](cluster-jmx-management.md)
+  - [Rolling Updates](rolling-updates.md)
```

Comment:
Is this the right place on the side menu? should it be the last one? Or closer to bootstrap?